src/main/java/view/panels/PortfolioPanel.java

```diff
@@ -35,20 +50,8 @@ public PortfolioPanel() {
         add(tableScrollPane, BorderLayout.CENTER);
     }
 
-    public static void main(String[] args) {
-        JFrame frame = new JFrame("Portfolio Panel");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setSize(800, 600);
-
-        PortfolioPanel portfolioPanel = new PortfolioPanel();
-        frame.add(portfolioPanel);
-
-        frame.setVisible(true);
-    }
-
     private JTable createPortfolioTable() {
```

Comment:
You can remove createPortfolioTable and getDefaultTableModel now that you're using real portfolio data.